### ISSUE TITLE
New Update

### ISSUE THREAD

**Issue by mmitana** [al-caughey/Google-Meet-Attendance#124]
The newest update has made it so the extension is glitching. Only the top bar is visible and it is in a fixed position on the upper left corner of the meet. Several other teachers on my platform are having the same issue.

**Comment by jcathhs** [comment]
Ditto - I did see a random  message about an update being succesful and being returned to a developer console.

**Comment by bythmark** [comment]
I'm having the same issues.

Something's odd about this. In Chrome, the extension also pops up with a full-screen ad to a different "v2" extension advertising a "safe screen sharing" feature, which seems oddly out of scope, and the page has a different developer account and contact information. The extension's functionality appears to be completely changed to get you to go add this new, different extension with very little explanation. There's also no link to the facebook page where information on updates like this is usually posted. I have not added the new extension and have removed this one after the update.

This repo hasn't been updated since March. The last post on the facebook page from 5/14 does have a kind of goodbye feel to it, I guess. Was development handed off to someone else?

Old extension: https://chrome.google.com/webstore/detail/google-meet-attendance/fkdjflnaggakjamjkmimcofefhppfljd?hl=en

New(?) Extension: https://chrome.google.com/webstore/detail/google-meet-attendance-v2/hhdmekoddjmghpckcgejaekaoelfbjhp

**Comment by igtbtk8** [comment]
I agree something is weird. The support e-mails are different on the two extensions. I removed the extension for now, because I'm considering it untrusted until I see an update from Al about this. I don't remember the app requiring such broad permissions before. sandrazeilinger1985@gmail.com is the contact e-mail on the "old extension" which I'm sure it's possible that Al uses a different e-mail from his name, it's a bit strange that you have what looks like a person e-mail for a different person on there.

**Comment by 1erXeneize** [comment]
Same here. I wrote to the weird email that supposedly is the programmer but no answer yet.
What was strange was that the extension updated automatically without asking for permissions, etc. 
I uninstalled until further notice from original programmer. BTW, there is nothing in Al's youtube channel about this update.
We in Argentina are in fully virtual mode yet, so, this tool is essential to us. Hope it gets clarified soon.
Need to check on Facebook yet.
Cheers

**Comment by lmcalpine** [comment]
Having same issue.  This was my go-to extension for attendance this year.  Sadly, I will be uninstalling since new "safe sharing screen" renders the extension totally unusable.

**Comment by JonavarrCETI** [comment]
I'm having the same problem here. The extension looks hanged. I have no access to the attendance lists. Maybe they're gone

**Comment by knichel** [comment]
Ditto.  same issue with only the top bar showing.  v2 is blocked by administration (not whitelisted) and it takes an act of congress to get new extensions approved.  Attendance is now rendered useless for me.

**Comment by 1erXeneize** [comment]
This is what I received as answer email from developer:

""""
Hello and thank you for your mail,

We have released a patch and are awaiting its release shortly.

We are very sorry for the inconvenience.

Best regards,
Google Meet Attendance Team
""""

**Comment by mmitana** [comment]
> I'm having the same issues.
> 
> Something's odd about this. In Chrome, the extension also pops up with a full-screen ad to a different "v2" extension advertising a "safe screen sharing" feature, which seems oddly out of scope, and the page has a different developer account and contact information. The extension's functionality appears to be completely changed to get you to go add this new, different extension with very little explanation. There's also no link to the facebook page where information on updates like this is usually posted. I have not added the new extension and have removed this one after the update.
> 
> This repo hasn't been updated since March*. The last post on the facebook page from 5/14 does have a kind of goodbye feel to it, I guess. Was development handed off to someone else?
> 
> Old extension: https://chrome.google.com/webstore/detail/google-meet-attendance/fkdjflnaggakjamjkmimcofefhppfljd?hl=en
> 
> New(?) Extension: https://chrome.google.com/webstore/detail/google-meet-attendance-v2/hhdmekoddjmghpckcgejaekaoelfbjhp
> 
> *edit: both of the extensions linked were updated on May 26

**Comment by mmitana** [comment]
I added the "new" extension (v2). It is working for me. I used the downloads from yesterday to upload my class lists.

**Comment by igtbtk8** [comment]
I got that same e-mail. I'm kind of suspicious though, because the name on that e-mail does not match the name of the developer on Github, and who has been releasing videos for the past year. It also has text written in German in the quoted email, whereas the developer is based in Canada, according to other information. Perhaps Al works with this person, who is based in a German speaking country? It always seems like he has been a solo show. I don't trust the v2 extension, and I won't be downloading it.

**Comment by 1erXeneize** [comment]
> I added the "new" extension (v2). It is working for me. I used the downloads from yesterday to upload my class lists.

hi @mmitana, How did you do it? I tried V2 but still unusable.

**Comment by 1erXeneize** [comment]
> I got that same e-mail. I'm kind of suspicious though, because the name on that e-mail does not match the name of the developer on Github, and who has been releasing videos for the past year. It also has text written in German in the quoted email, whereas the developer is based in Canada, according to other information. Perhaps Al works with this person, who is based in a German speaking country? It always seems like he has been a solo show. I don't trust the v2 extension, and I won't be downloading it.

It is suspicious agree. Not sure what to do now, so perhaps, wait and see what happens.

**Comment by AlCaughey** [comment]
Everyone -  The Google Meet Attendance extension has been purchased by
another organization and I am no longer involved with the project... sorry,
I am not allowed to discuss the details any further than that.  Rest
assured it was not an easy decision!

I have told them about the issues in version 1.5.  Support questions should
now be sent to ***@***.***

Sorry!  Al

**Comment by JonavarrCETI** [comment]
Thanks for your kind answer!

**Comment by Eagle-Man** [comment]
@1erXeneize Disable/uninstall the old one once the new one is installed.  They will conflict with each other.

**Comment by JonavarrCETI** [comment]
Thank you! ☺

**Comment by JonavarrCETI** [comment]
Is the one with the version 1.5 the new one?

**Comment by Eagle-Man** [comment]
No.  The new one is called "Google Meet Attendance - v2".  Not sure if they're trying to keep both running or if they're trying to switch it to the new one.

**Comment by JonavarrCETI** [comment]
I think I'm confused, From where should I download "Google Meet-Attendance
v2"?

**Comment by AlCaughey** [comment]
The Google Meet Attendance extension has been purchased by another
organization and I am no longer involved with the project... sorry, I am
not allowed to discuss the details any further than that.  Rest assured it
was not an easy decision!

I have told the new developers about the issues in version 1.5.  Support
questions should now be sent to them at ***@***.***

Thank you so much for your interest in this project.

Sorry!  Al

**Comment by JonavarrCETI** [comment]
You did a great job. Your contribution won't be forgotten.

I wish you to succeed in your future projects.
